src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -653,7 +654,7 @@ private void removePlayerClicked(final Button button)
         if (row >= 0 && row < users.size())
         {
             final Permissions.Player user = users.get(row);
-            if (user.getRank() != Permissions.Rank.OWNER)
+            if (user.getRank() != Rank.OWNER)
```

Comment:
I think this belong to a different pr, and could be merge well before dman 2.0 is ready